### PR TITLE
Enable proposal for multiple output

### PR DESCRIPTION
**Pull request by kelvinchoi-intel**

### CODE REVIEW

File: src/plugins/intel_gpu/src/graph/include/activation_inst.h [hunk 1]
```diff
@@ -24,6 +24,7 @@ struct typed_program_node<activation> : public typed_program_node_base<activatio
     using parent::parent;
 
     program_node& input() const { return *get_dependency(0).first; }
+    int32_t input_idx() const { return get_dependency(0).second; }
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/include/activation_inst.h:27]: Ummm it sounds like the index of input within this node, but it is pointing the output idx of the previous node for the corresponding input..... Could we come up with better naming for this method to be more intuitive?
For example, input_dep_idx, etc... thought its still blury.. :( (I am not good at naming though)

**Comment by yeonbok** [comment]
Please rebase this.

File: src/plugins/intel_gpu/src/graph/include/activation_inst.h (hunk 1 above)
Comment (kelvinchoi-intel) [reply src/plugins/intel_gpu/src/graph/include/activation_inst.h:27]: How about "output_idx_of_input_node()" or "input_node_output_idx()"?

Comment (yeonbok) [reply src/plugins/intel_gpu/src/graph/include/activation_inst.h:27]: hmmm how about input_dep_idx_to_prev_node ?

Comment (kelvinchoi-intel) [reply src/plugins/intel_gpu/src/graph/include/activation_inst.h:27]: I've changed to 'input_dep_idx'.

**Pull request merged**